src/test/kotlin/no/fdk/dataset_catalog/utils/TestData.kt

```diff
@@ -151,9 +151,27 @@ private fun Dataset.mapDBO(): org.bson.Document =
         .append("catalogId", catalogId)
         .append("title", title)
         .append("description", description)
+        .append("references", references?.map { it.mapDBO() })
         .append("registrationStatus", registrationStatus.toString())
 
 private fun Catalog.mapDBO(): org.bson.Document =
     org.bson.Document()
         .append("_id", id)
         .append("uri", uri)
+
+private fun Reference.mapDBO(): org.bson.Document =
```

Comment:
Det er disse 3 som gjør at integrasjonstestene feiler. Siden du ikke faktisk tester references i integrasjonstestene kan du jo bare fjerne min commit. 

Eventuelt så blir det å oppdatere references for datasettene i `datasetDbPopulation`, slik at koden ikke prøver å legge til strenger med whitespace i uri'er.

Reply:
Jeg fjernet mellomrom og oppdaterte expected turtle fila.